config/settings.yml

```diff
@@ -639,6 +639,15 @@ claims_api:
     enabled: false
   token_validation:
     api_key:
+  benefits_documents:
+    auth:
+      ccg:
+        client_id: ~
+        rsa_key: ~
+    host: https://sandbox-api.va.gov
+    access_token:
+      aud_claim_url: https://deptva-eval.okta.com/oauth2/ausi3ui83fLa68IJv2p7/v1/token
```

Comment:
doesn't this want to live within the `benefits_documents.auth.ccg`?

Reply:
yes it does, I wanted to wait to make sure the secrets stuff was going to stay the same.  I will update this now